controllers/velero.go

```diff
@@ -144,6 +146,27 @@ func (r *DPAReconciler) veleroClusterRoleBinding(dpa *oadpv1alpha1.DataProtectio
 	return crb, nil
 }
 
+// Apply Override unsupported Args
+func (r *DPAReconciler) applyUnsupportedArgsOverride(dpa *oadpv1alpha1.DataProtectionApplication, veleroDeployment *appsv1.Deployment, veleroContainer *corev1.Container) error {
+	if configMapName, ok := dpa.Annotations[UnsupportedVeleroServerArgsAnnotation]; ok {
+		// Got an DPA annotation for Unsupported Server Args
+		if configMapName == "" {
+			// Can't be empty string, do not report error
+			return nil
+		}
+
+		unsupportedServerArgsCM := corev1.ConfigMap{}
+		if err := r.Get(r.Context, types.NamespacedName{Namespace: dpa.Namespace, Name: configMapName}, &unsupportedServerArgsCM); err != nil {
+			return err
```

Comment:
For non-existing config map I get the following error:
```
  conditions:
  - lastTransitionTime: "2024-06-21T16:18:21Z"
    message: ConfigMap "non-existing-cm" not found
    reason: Error
    status: "False"
    type: Reconciled

```

Should we add more color to the error message ? maybe something like `Config <name> for unsupported velero server args not found` ? Current error also looks good though, up to you.

Reply:
I prefer not to mess up with standard error messages if they are good enough, so I will leave it.